vignettes/phyloVision.Rmd

```diff
@@ -43,28 +56,86 @@ library(ape)
 
 ## Creating a PhyloVision Object with a Tree
 
-Vision objects now support dendrograms for visualization and analysis. To create the PhyloVision object, you need all of the same data, an expression matrix, and metadata and/or signatures, but you also include a tree object from the Ape package.
+We'll proceed by reading in the raw expression matrix, tree, and meta data for a single clone (CP007) of approximately 600 cells from [Quinn et al, Science 2021](https://www.science.org/doi/abs/10.1126/science.abc1944).
 
-```{r create, eval=F}
-# Read the expression and meta data
-file_path <- "data/embryogenesis" # Replace the path
+```{r, eval=F}
+# Read in expression data
+expression <- read.table('data/metastasis/cp007_expression.tsv.gz', sep='\t',
+                         header=T, row.names=1, check.names = FALSE,
+                         stringsAsFactors = F)
+
+# Read in tree
+tree <- read.tree('data/metastasis/cp007_tree.nwk.gz')
```

Comment:
I get a newick format error when trying to run the hotspot commands later on, I think this is happening because the cp007 newick has a ":1" appended to the end of leaf names.

Reply:
These are branch lengths (specifically, every branch is the same length). What's the error exactly?

Reply:
I believe I fixed it. Ete3 gets really finicky about the format of the newick string, so setting `format=1` fixes it because it knows to look for branch lengths. Unless was there a specific reason you had `format=8L`?

Reply:
I set it as 8 for both internal names and leaf names, I think 1 might work since it's flexible and ete says "
flexible with internal node names | ((D:0.723274,F:0.567784)E:0.067192,(B:0.279326,H:0.756049)B:0.807788), Formats labeled as flexible allow for missing information.;" but maybe this should be an argument to the function?